src/Api/Middlewares/ExceptionHandler.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using Domain.Exceptions;
+using Microsoft.EntityFrameworkCore;

```

Comment:
Por que usou EF aqui?

Reply:
Essa using já existia ele é necessário para as condicionais  if (ex is DbUpdateException) e  if (ex is DbUpdateConcurrencyException)